ai-integrations/haystack.ipynb

```diff
@@ -137,7 +144,7 @@
     "]\n",
     "\n",
     "# Initializing a document embedder to convert text content into vectorized form.\n",
-    "doc_embedder = OpenAIDocumentEmbedder()\n",
+    "doc_embedder = VoyageDocumentEmbedder()\n",
```

Comment:
This initialization is failing with the error that the auth environment variable named `'VOYAGE_API_KEY'` is not set - I think we need to rename the var up above because this is expecting it to have this specific name.